gettsim/demographic_vars.py

```diff
@@ -0,0 +1,18 @@
+def _anz_kinder_in_tu(tu_id, kind):
```

Comment:
Does it make sense to combine this module with the one in alg2? just to have eveything in one place?

Reply:
Well, its not only an alg2 issue. And for further use. I just suggest, that we keep this kind of variables in a separate place!